x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferencePlugin.java

```diff
@@ -203,6 +203,12 @@ public class InferencePlugin extends Plugin
         License.OperationMode.ENTERPRISE
     );
 
+    public static final LicensedFeature.Momentary EIS_INFERENCE_FEATURE = LicensedFeature.momentary(
+        "inference",
+        "eis",
```

Comment:
Its a bit confusing to me, the `ElasticInferenceService.NAME` is `elastic`, but our license service name is `eis`. So the user configures `elastic` to get the license with name `eis`. 

Could we name this `elastic` to match the name of the `ElasticInferenceService.NAME`?

Reply:
I've changed this to the proper name "Elastic Inference Service"

The error the user sees if the licence is not compatible is either:
```
"current license is non-compliant for [inference]"
```

Or 

```
"current license is non-compliant for [Elastic Inference Service]"
```